content/adopting-an-ethical-license.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Adopting an Ethical License"
+heading = "Adopting an Ethical License"
++++
+
+If you are starting a new project, you _may_ [review existing ethical licenses](./licenses/) and choose the one that most directly matches your goals.
+
+For maintainers considering adopting an ethical license on an existing project, be aware that this requires significant work. Each license is different, which makes a general guide difficult to write. Bear in mind that this is not legal advice, and we are not lawyers.
+
+Because there is significant social risk associated with relicensing, we recommend starting by centering an intent to preserve and further develop the trust between maintainers, contributors, and users of the project.
+
+From there, an understanding of the authorship rights and license constraints will serve maintainers well. To do so, we recommend reaching a high degree of confidence in the precision and accuracy of your answers to the following questions:
+
+1. Who holds the authorship rights?
+2. What constraints does the current license impose upon distributors or derivative works?
+
+## A Potentially Useful Framing
+
+An analogy that may (or may not!) be useful as you move forward is to think about intellectual property rights in the same manner of other property rights. Let's pretend a project is a "neighborhood." Each "house" has a "lock" on it that the "homeowner" installed when it was "built". This "lock" is opened with the right "key." If, as the "neighborhood maintainer", we want to change the lock, we _must_ check with the homeowner; unless they have given prior consent.
+
+The "homeowner" is the contributor. The "house" is a substantial contribution. The "lock" is the license, and the constraints the license imposes is the "key" to granting you the right to use the home.
+
+## Authorship Rights
+
+Authorship rights are often held by the original author of the contributed source code. Exceptions exist, such as when contributors explicitly assign authorship to the person(s) or organization(s) that maintain the project through a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement).
```

Comment:
This is a little inaccurate. Possible rewrite:

## Authorship Rights

By default, the original author of the contributed source code owns copyright on the expressive content in that code. Copyright is the right to exclude others from certain activities relating to that expressive content, and is the main legal basis for software licensing. (Other kinds of intellectual property rights, such as patents, are also important in software licensing, but for simplicity we'll ignore them for purposes of this guide.) A license is a grant of permission to others to exercise rights that the owner could otherwise restrict. Typically, an employer owns copyright on software created by an employee. 

In a collaborative community software project, it is generally understood that each contributor is licensing their copyright under the license selected by the project. Exceptions exist, such as when a project requires contributors to sign a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) requiring them to license copyright under more generous terms than the project license itself would require.

Reply:
Thank you! This is a helpful reframing!

Reply:
> In a collaborative community software project, it is generally understood that each contributor is licensing their copyright under the license selected by the project. Exceptions exist, such as when a project requires contributors to sign a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) requiring them to license copyright under more generous terms than the project license itself would require.

@richardfontana, I think that most of the times, even when a CLA is present, contributions are still licensed under the same license as the project (to anybody receiving them) and, *in addition* they are licensed to a specific party according to the CLA (the contribution is effectively dual-licensed to them). Obviously that depends on the CLA but I think this is the most common scenario.

In practice it may not matter a lot, but I think it might be slightly more accurate?